yew-router/examples/minimal/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+../run_example.sh minimal
```

Comment:
Useful having a new line at the end :)
It makes POSIX happy
```suggestion
../run_example.sh minimal

```

Reply:
Yea, it's really strange, this file was created on Linux too :P not really sure what happened